torch/__init__.py

```diff
@@ -490,15 +489,29 @@ def sym_min(a, b):
         return b.__sym_min__(a)
     return builtins.min(a, b)  # type: ignore[operator]
 
-# Drop in replacement for math.sqrt
-def sym_sqrt(a):
```

Comment:
Unlike `sym_int` and `sym_float`, I don't think ever any reason you would need this this instead of `math.sqrt` so I think it should be okay to drop this.

Reply:
While I do think it's likely okay to get rid of `sym_sqrt`, by the same token this PR would probably be easier to merge if we just add `sym_sqrt = _sym_sqrt` in this file.